File: LegoSCart/asplego/Models/Checkout.cs
using System;
namespace Asplego.Models
{
    public class Checkout
    {

        public string userEmail { get; set; }
        public Lego list { get; set; }
        public int count { get; set; }


        public Checkout(string userEmail1, Lego list1,int count1)
        {

            userEmail = userEmail1;
            list = list1;
            count = count1;


        }
    }


}

File: LegoSCart/LegoShop/src/app/shared/models/Checkout.ts
import {Lego} from './Lego';

export class Checkout{
    userEmail:string;
    list:Lego;
    count:number;

    constructor(userEmail1:string, list1:Lego,count1:number){
        this.userEmail=userEmail1;
        this.list=list1;
        this.count=count1;

    }
}